src/Libraries/Microsoft.Extensions.AI.Abstractions/Contents/DataContent.cs

```diff
@@ -167,10 +175,9 @@ public string Uri
     /// If the media type was explicitly specified, this property returns that value.
     /// If the media type was not explicitly specified, but a data URI was supplied and that data URI contained a non-default
     /// media type, that media type is returned.
-    /// Otherwise, this property returns null.
     /// </remarks>
-    [JsonPropertyOrder(1)]
-    public string? MediaType { get; private set; }
+    [JsonIgnore]
```

Comment:
Is this because the JSON representation is meant to be just the data URI which should include this information anyway?

Reply:
Yes